src/torchmetrics/functional/detection/panoptic_quality.py

```diff
@@ -41,6 +41,7 @@ def panoptic_quality(
     where IOU, TP, FP and FN are respectively the sum of the intersection over union for true positives, the number of
     true postitives, false positives and false negatives. This metric is inspired by the PQ implementation of
     panopticapi, a standard implementation for the PQ metric for object detection.
+    The metric was introduced in https://arxiv.org/abs/1801.00868.
```

Comment:
```suggestion
    The metric was introduced in `Panoptic Segmentation paper`_.
```
again add https://arxiv.org/abs/1801.00868 to https://github.com/Lightning-AI/metrics/blob/master/docs/source/links.rst